docs/howto/blueprints.rst

```diff
@@ -0,0 +1,24 @@
+Use Blueprints
+--------------
+
+You can use blueprints to break up your tasks.
+
+Firstly use blueprints to create a task collection::
+
+    from procrastinate import blueprints
```

Comment:
Anything that is to be accessed from outside of the project is aliased in the `procrastinate` namespace, so we'll need to add it to `procrastinate/__init__.py` and the doc can either:
```
import procrastinate
procrastinate.BluePrint()  # (preferred way)

# or

from procrastinate import BluePrint
BluePrint()
```